python/lib/metrics/dmod/test/test_metadata.py

```diff
@@ -17,7 +17,7 @@ def test_metadata_retrieval(self):
         """
         found_metrics = metrics.get_all_metrics()
 
-        self.assertEqual(len(found_metrics), 12)
+        self.assertEqual(len(found_metrics), 13)
```

Comment:
I see this test fail, seemingly expecting 14.

Reply:
It wasn't accounting for `ProbabilityOfFalseDetection` and the new `LinearTemporalTrendAbsoluteError`. I'm fixing the number and adding the missing checks.